Solutions/Lucas/ex_1/handlers/input_sqs.py

```diff
@@ -0,0 +1,36 @@
+import json
+from   flask import request, Response
+from   src import data_validation, aws_utility
+
+
+def handle():
+    print('---------------------------------------------')
```

Comment:
Sempre que possível usar o pacote [loggin](https://docs.python.org/3/howto/logging.html) do python.

ref: https://towardsdatascience.com/stop-using-print-and-start-using-logging-a3f50bc8ab0
ref: https://docs.python.org/3/howto/logging.html
ref: https://stackoverflow.com/questions/6918493/in-python-why-use-logging-instead-of-print